sale_commission/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -8,3 +8,36 @@ def migrate(env, version):
     openupgrade.load_data(
         env.cr, "sale_commission", "migrations/13.0.1.0.0/noupdate_changes.xml"
     )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_invoice_line_agent aila
+        SET object_id = aml.id
+        FROM account_move_line aml
+        WHERE aml.old_invoice_line_id = aila.{}
+    """.format(
```

Comment:
Please use `sql.SQL` and `sql.Identifier` for proper SQL build.

Reply:
Sure!